tasks/src/reinforcement-learning/about.md

```diff
@@ -25,13 +18,64 @@ This place can be filled with variants of this task if there's any.
+
+Dialogue systems use reinforcement learning because of its inherent nature of decision-making. It learns from mistakes. The system tries to understand the sentence's state and form an action set maximizing the value it would add. Reinforcement Learning is used in various NLP tasks such as text summarization, question answering, translation, dialogue generation, or machine translation. Reinforcement Learning agents can be trained to understand a few sentences of the document and use it to answer the corresponding questions.
+
+Robots in industries or healthcare working towards reducing manual intervention use reinforcement learning to map natural language instructions to sequences of executable actions. During training, the learner repeatedly constructs action sequences, executes those actions, and observes the resulting rewards. A reward function works in the backend that defines the quality of these executed actions.
+
+### Robotics 
+
+Robots operate in a highly dynamic and ever-changing environment, making it impossible to predict what will happen next. Reinforcement Learning provides a considerable advantage in these scenarios to make the robots robust enough and help acquire complex behaviors adaptively in different scenarios. It aims to remove the need for time-consuming and tedious checks and replaces them with computer vision systems ensuring higher levels of quality control on the production assembly line.
+
+A wide variety of problems in robotics can be considered problems related to reinforcement learning. Reinforcement learning enables a robot to autonomously discover an optimal behavior by interacting with its environment.
+
+## Task Variants 
+
+### Q - Learning
+
+Q - learning will find the best course of action, given the agent's current state. Depending on where the agent is in the environment, it will decide the following action to be taken. It chooses the action such that it maximizes the reward.
+
+#### Sequential decision making problem for text understanding
+
+Agent observes state as a string of text at a time 't'. Agent also knows a set of possible actions, each describes as a string text. Agent tries to understand the “state text” and all possible “action texts”, then takes a decision which maximizes the long term reward. Then, the environment state changes to a new state, agent receives an immediate reward.
+
+#### Unbounded action space in reinforcement learning
+
+Environments with vast state space is a problem in reinforcement learning. Also the action space can be quite large. 
+
+For solving this problem DRRN(deep reinforcement relevance network) was proposed. The idea of DRRN is to project both the state and action into a continuous space(as vectors). Q-function is a relevance function of the state vector and action vector.the DRRN experiences higher average reward.

```

Comment:
```suggestion
To solve this problem, DRRN (deep reinforcement relevance network) was proposed. The idea of DRRN is to project both the state and action into a continuous space (as vectors). Q-function is a relevance function of the state vector and action vector. The DRRN experiences a higher average reward.

```
If we are going to have this can you please explain more how it works?

Reply:
It's the first time I hear about relevance network. I was wondering if it's something "too advanced for the task page? 🤔